pages/services/index.vue

```diff
@@ -1,15 +1,12 @@
 <template>
-  <div>
-    <HeroBlock hero-classes="services">
+  <div itemscope itemtype="http://schema.org/Service">
```

Comment:
I don't think we want this `itemscope itemtype` stuff for this page. It's generally for a specific service, and provides the metadata to tell google about that service.